test/libsolidity/syntaxTests/license/license_hidden_unicode.sol

```diff
@@ -0,0 +1,3 @@
+// This is parsed as GPL-3.0:
+// SPDX-License-Identifier: GPL-3.0 ⡉⡊⡋⡌⡍⡎⡏⡐⡑⡒
```

Comment:
Basically can't avoid creating metadata tests to compare what goes into the output 😢